### DISCUSSION TITLE
Allow Admin to More Easily Make Folks into Tag Admins

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#20306] [Feature Requests]
Currently, we require admin to copy a user's id or username and then go find the tag page in the admin, where we can add the mod to the tag.

I think it'd be easier to be able to go to a user's admin page and add the tag to the user from that interface.

We have a tag mod application and often folks request to be mods for multiple tags. It would be a much quicker process if admin could add several tags to a single mod, instead of having to search and click into each different tag to add the mod to it.

Here's a recent bug report video I filmed that shows the current process:

https://www.loom.com/share/3b791c43a3834e3f970bd9d0481cb77b?sid=e3f15dd2-a43e-49f5-a34e-fbaac9d0d2ce

... you can imagine that this could be clunky if I'm meant to add a single person to several tags. It would be much more efficient to give admin a space in the user's admin page (e.g. https://dev.to/admin/member_manager/users/1186667) to add the tags to that user. This would work similarly to how you can add a user to and org from the user's page. (see pic below)

![Screen Shot 2023-11-02 at 10 59 19 AM](https://github.com/forem/forem/assets/16007075/48c92c66-7342-4b34-9f68-0bae1dd30784)

**Comment by Justman100** [comment]


**Reply by michael-tharrington** [reply]
Heyo! Please only respond with comments related to this issue. The issue you're linking to is unrelated. I'm going to delete this comment. Hope you understand!
